cabal-install/tests/UnitTests/Options.hs

```diff
@@ -18,14 +19,15 @@ extraOptions :: [OptionDescription]
 extraOptions =
   [ Option (Proxy :: Proxy OptionShowSolverLog)
   , Option (Proxy :: Proxy OptionMtimeChangeDelay)
+  , Option (Proxy :: Proxy RunNetworkTests)
   ]
 
 newtype OptionShowSolverLog = OptionShowSolverLog Bool
   deriving Typeable
 
 instance IsOption OptionShowSolverLog where
   defaultValue   = OptionShowSolverLog False
-  parseValue     = fmap OptionShowSolverLog . safeRead
+  parseValue     = fmap OptionShowSolverLog . safeReadBool
```

Comment:
Unrelated change?

Reply:
I guess, but it keeps consistency between OptionShowSolverLog and RunNetworkTests which are both bool type options.